coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java

```diff
@@ -544,6 +545,19 @@ public boolean areActiveQuestionnaires(List<AnswerHeader> answerHeaders) {
         return false;
     }
 
+    public boolean areAnsweredQuestionnaires(List<AnswerHeader> answerHeaders) {
+        for (AnswerHeader answerHeader : answerHeaders) {
+            if (answerHeader.isActive()) {
+                for (Answer answer : answerHeader.getAnswers()) {
+                    if (StringUtils.isNotEmpty(answer.getAnswer())) {
```

Comment:
I think this is wrong.  Don't all questions have to be answered (ie all answer objects have a non-blank answer).  Also is there existing code you can reuse that determines if a questionaire is complete?